apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -10,7 +10,10 @@
   }
 
   video {
-    display: none;
+    visibility: hidden;
```

Comment:
Properties should be ordered position, width, height, visibility

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)